<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONNXExport</class>
 <widget class="QDialog" name="ONNXExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NX Server Copy DLL - Visionin</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Path</string>
   </property>
   <widget class="QComboBox" name="comboBox_path_in">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>27</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dir_in">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>26</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_xport">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy DLL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Plugin DLL Path</string>
   </property>
   <widget class="QComboBox" name="comboBox_path_out">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>27</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dir_out">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>26</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
